drivers/staging/android/lowmemorykiller_tng.c

```diff
@@ -161,8 +161,8 @@ static void calc_params(struct calculated_params *cp, gfp_t mask)
 	int i;
 	int array_size;
 	long free_swap = get_nr_swap_pages();
-	long irp = global_node_page_state(NR_INDIRECTLY_RECLAIMABLE_BYTES)
-	  >> PAGE_SHIFT;
+	long irp = global_node_page_state(NR_SLAB_RECLAIMABLE) +
+            global_node_page_state(NR_KERNEL_MISC_RECLAIMABLE);
```

Comment:
Here you are using this conversion: https://github.com/sonyxperiadev/kernel/commit/b29940c1abd7a4c3abeb926df0a5ec84d6902d47#diff-cef95765dfd76e5f9c9f0faebfa683edf904d0c3de71547ae8c3ea14418c1e38R4734